opentelemetry-sdk/src/logs/mod.rs

```diff
@@ -83,14 +82,19 @@ mod tests {
         let attributes: Vec<(Key, AnyValue)> = log
             .record
             .attributes
-            .clone()
-            .expect("Attributes are expected");
+            .iter()
```

Comment:
isn't this change fully internal only? Why we are forced to change processors to account for this?